fw/http_frame.c

```diff
@@ -1010,7 +1010,7 @@ tfw_h2_wnd_update_process(TfwH2Ctx *ctx)
 fail:
 	if (!ctx->cur_stream) {
 		tfw_h2_conn_terminate(ctx, err_code);
-		return T_BLOCK;
+		return T_BAD;
```

Comment:
Please pay attention may be you have idea about more appropriate error code from errno.h